core/src/player.rs

```diff
@@ -1302,7 +1302,7 @@ impl Player {
                     );
                     if let Ok(prototype) = constructor.get("prototype", &mut activation) {
                         if let Value::Object(object) = actions.clip.object() {
-                            object.set_proto(activation.context.gc_context, prototype);
+                            object.set_proto(&mut activation, prototype).unwrap(); // TODO: What happens on error?
```

Comment:
Don't think there's any actual way for this to error, as a watcher would be the only possibility to return an error, and it should be impossible for the object to have a watcher at this point.